src/components/HelloWorld.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <div id="container">
+      <h1>Menu of Salads</h1>
+      <ul>
+        <li v-for="(item, index, price) in myJson" :key="index" :id="index">
+          {{ index + 1 }}. {{ item }}
+          <div class="lisupport">
+            <p v-if="index % 3 == 0">{{ (price = +15) }}</p>
```

Comment:
Fajnie, że skorzystałeś z arytmetyki modulo. Właśnie to chciałem zobaczyć :) !

Reply:
Tutaj to nie ma znaczenia, ale staraj się zawsze używać silnego porównania '==='. Jeżeli nie wiesz jaka jest różnica między == a === odsyłam do googla :P